tools/serverpod_cli/lib/src/analyzer/dart/endpoint_analyzers/endpoint_class_analyzer.dart

```diff
@@ -40,11 +41,25 @@ abstract class EndpointClassAnalyzer {
     return true;
   }
 
-  /// Validates the [ClassElement] and returns a list of errors.
+  /// Validates the [ClassElement] and [endpointDefs] and returns a list of errors.
```

Comment:
Suggestion: Revert description change.

We only validate the `ClassElement`, the `endpointDefs` are only there to support that validation and are not actually validated.